src/rebar_ct.erl

```diff
@@ -288,7 +288,7 @@ search_ct_specs_from(Cwd, TestDir, Config) ->
 build_name(Config) ->
     %% generate a unique name for our test node, we want
     %% to make sure the odds of name clashing are low
-    Random = integer_to_list(crypto:rand_uniform(0, 10000)),
+    Random = integer_to_list(rand:uniform(10000)),
```

Comment:
Technically the ranges here might be 1 off but not sure how much that matters in practice. For instance, `rand:uniform/1` won't produce 0s its `[1, Hi]` range so `[1, 10000]`. rand_uniform looks like produces values in `[Lo, Hi)` range so `[0, 10000)`

I don't think it really matters here but it's worth noting just in case